src/network/assert.rs

```diff
@@ -37,7 +34,7 @@ fn assert_or_revert_bool_with<E: FnOnce() -> Error>(check: bool, error: E) {
 
     let error = error();
     print_debug!("REVERT({}) - {}!", &error.code(), error);
-    revert(error);
+    panic!("{}", error);
```

Comment:
I wanted to have here a platform-dependent `revert` ;)

Reply:
I have left this part + print-debug in not so great state :D. If you are fine with this approach it will be fixed in future work :D